invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/DocumentRequests/DocumentRequestForm/DocumentRequestForm.js

```diff
@@ -32,111 +31,88 @@ export default class DocumentRequestForm extends Component {
-        onSubmit={this.onSubmit}
-        render={({ handleSubmit, isSubmitting }) => (
-          <Form
-            id="document-request-form"
-            onSubmit={handleSubmit}
-            loading={isSubmitting}
-          >
-            <StringField
-              fieldPath="title"
-              label="Title"
-              placeholder="Title"
-              required
-            ></StringField>
-            <StringField
-              fieldPath="authors"
-              label="Authors"
-              placeholder="Authors"
-            ></StringField>
-            <Form.Group widths="equal">
-              <StringField
-                fieldPath="isbn"
-                label="ISBN"
-                placeholder="ISBN"
-              ></StringField>
-              <StringField
-                fieldPath="issn"
-                label="ISSN"
-                placeholder="ISSN"
-              ></StringField>
-            </Form.Group>
-            <Form.Group widths="equal">
-              <StringField
-                fieldPath="volume"
-                label="Volume"
-                placeholder="Volume number"
-              ></StringField>
-              <StringField
-                fieldPath="issue"
-                label="Issue"
-                placeholder="Issue number"
-              ></StringField>
-              <StringField
-                fieldPath="page"
-                label="Page"
-                placeholder="Page number"
-              ></StringField>
-              <StringField
-                fieldPath="publication_year"
-                label="Publication Year"
-                placeholder="Publication Year"
-              ></StringField>
-            </Form.Group>
-            <TextField
-              fieldPath="note"
-              label="Note"
-              placeholder="Notes for the library"
-              uiProps={{ rows: 5 }}
-            ></TextField>
-
-            <Button type="submit" disabled={isSubmitting}>
-              Request Book
-            </Button>
-          </Form>
-        )}
-      />
+        submitSerializer={this.onSerializeSubmit}
+        successCallback={this.onSubmitSuccess}
+        successSubmitMessage="Your book request has been sent to the library."
+        createApiMethod={documentRequestApi.create}
+      >
+        <StringField
+          fieldPath="title"
+          label="Title"
+          placeholder="Title"
+          required
+        />
+        <StringField
+          fieldPath="journal_title"
+          label="Journal title"
+          placeholder="Journal title"
+        />
+        <StringField
+          fieldPath="authors"
+          label="Authors"
+          placeholder="Authors"
+        />
+        <Form.Group widths="equal">
+          <StringField fieldPath="isbn" label="ISBN" placeholder="ISBN" />
+          <StringField fieldPath="issn" label="ISSN" placeholder="ISSN" />
+        </Form.Group>
+        <Form.Group widths="equal">
```

Comment:
Just food for thought: Do you think it's worth it to make component like `GroupField` that just wraps the components in a `Form.Group` ? Like:
```suggestion
        <GroupField uiProps={{width: "equal"}}>
          <StringField fieldPath="isbn" label="ISBN" placeholder="ISBN" />
          <StringField fieldPath="issn" label="ISBN" placeholder="ISSN" />
        </GroupField
```